source/application/AMAppController.cpp

```diff
@@ -287,50 +298,20 @@ void AMAppController::openScanInEditor(AMScan *scan, bool bringEditorToFront, bo
 		mw_->setCurrentPane(editor);
 }
 
-#include "acquaman/AMScanConfiguration.h"
-#include "ui/acquaman/AMScanConfigurationView.h"
-#include "ui/acquaman/AMScanConfigurationViewHolder3.h"
-#include "dataman/database/AMDatabase.h"
-#include "dataman/database/AMDbObjectSupport.h"
-
 void AMAppController::launchScanConfigurationFromDb(const QUrl &url)
```

Comment:
Yeah, that is the main difference.  AMDatamanAppController shouldn't have a holder because you can't run anything.